# Adds threadsafe recreateDefault() method to ICU::Timezone.

diff --git a/intl/icu/source/i18n/timezone.cpp b/intl/icu/source/i18n/timezone.cpp
--- a/intl/icu/source/i18n/timezone.cpp
+++ b/intl/icu/source/i18n/timezone.cpp
@@ -573,16 +573,23 @@
 {
     umtx_initOnce(gDefaultZoneInitOnce, initDefault);
     {
         Mutex lock(&gDefaultZoneMutex);
         return (DEFAULT_ZONE != nullptr) ? DEFAULT_ZONE->clone() : nullptr;
     }
 }
 
+void
+TimeZone::recreateDefault()
+{
+    TimeZone *default_zone = TimeZone::detectHostTimeZone();
+    adoptDefault(default_zone);
+}
+
 // -------------------------------------
 
 TimeZone* U_EXPORT2
 TimeZone::forLocaleOrDefault(const Locale& locale)
 {
     char buffer[ULOC_KEYWORDS_CAPACITY] = "";
     UErrorCode localStatus = U_ZERO_ERROR;
     int32_t count = locale.getKeywordValue("timezone", buffer, sizeof(buffer), localStatus);
diff --git a/intl/icu/source/i18n/unicode/timezone.h b/intl/icu/source/i18n/unicode/timezone.h
--- a/intl/icu/source/i18n/unicode/timezone.h
+++ b/intl/icu/source/i18n/unicode/timezone.h
@@ -361,16 +361,19 @@
      *
      * @return   A default TimeZone. Clients are responsible for deleting the time zone
      *           object returned.
      * @see getUnknown
      * @stable ICU 2.0
      */
     static TimeZone* U_EXPORT2 createDefault();
 
+#define ICU_TZ_HAS_RECREATE_DEFAULT
+    static void U_EXPORT2 recreateDefault();
+
 #ifndef U_HIDE_INTERNAL_API
     /**
      * If the locale contains the timezone keyword, creates a copy of that TimeZone.
      * Otherwise, create the default timezone.
      *
      * @param locale a locale which may contains 'timezone' keyword/value.
      * @return   A TimeZone. Clients are responsible for deleting the time zone
      *           object returned.
